### PR TITLE
add a new node role for control plane

### LINKED ISSUE #242: use controlplane instead of master
https://github.com/kubernetes-csi/csi-driver-smb/blob/c16b61d150fa03d7ec0660bbd83e24e2955c1b97/charts/v0.6.0/csi-driver-smb/templates/csi-smb-controller.yaml#L23

### PR DESCRIPTION
**Pull request by boddumanohar**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
add a new node role for control plane

this was introduced in v1.20
https://kubernetes.io/docs/setup/release/notes/

The label applied to control-plane nodes "node-role.kubernetes.io/master" is now deprecated and will be removed in a future release after a GA deprecation period.

Introduce a new label "node-role.kubernetes.io/control-plane" that will be applied in parallel to "node-role.kubernetes.io/master" until the removal of the "node-role.kubernetes.io/master" label.

**Which issue(s) this PR fixes**:
Fixes #242 

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

File: charts/latest/csi-driver-smb/templates/csi-smb-controller.yaml [hunk 1]
```diff
@@ -27,6 +27,10 @@ spec:
           operator: "Equal"
           value: "true"
           effect: "NoSchedule"
+        - key: "node-role.kubernetes.io/controlplane"
```
Comment (andyzhangx) [inline charts/latest/csi-driver-smb/templates/csi-smb-controller.yaml:30]: also add this config in deploy/csi-smb-controller.yaml config

Comment (andyzhangx) [reply charts/latest/csi-driver-smb/templates/csi-smb-controller.yaml:30]: and what k8s version is this label used?

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: charts/latest/csi-driver-smb/templates/csi-smb-controller.yaml (hunk 1 above)
Comment (boddumanohar) [reply charts/latest/csi-driver-smb/templates/csi-smb-controller.yaml:30]: this was introduced in `v1.20`
https://kubernetes.io/docs/setup/release/notes/


* The label applied to control-plane nodes "node-role.kubernetes.io/master" is now deprecated and will be removed in a future release after a GA deprecation period.

* Introduce a new label "node-role.kubernetes.io/control-plane" that will be applied in parallel to "node-role.kubernetes.io/master" until the removal of the "node-role.kubernetes.io/master" label.

**Comment by andyzhangx** [comment]
pls also make similar change to nfs, blob csi drivers, thanks.

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**